### PR TITLE
feat : Login Page Added

### PR DESCRIPTION
**Pull request by vikrantkode**
# Added Login Page in e-comm
## HTML & CSS only

### CODE REVIEW

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]
PR and comment below it doesn't match. Didn't find any cart page in file change. PR raised should be for specific page you want to get reviewed. It will make reviewer's work easy.

File: Pages/Login/login.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Veekay Cart | Login</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link rel="stylesheet" href="https://veekay-cmplib.netlify.app/CSS/components.css">
+    <link rel="stylesheet" href="https://veekay-cmplib.netlify.app/CSS/header.css">
+    <link rel="stylesheet" href="https://veekay-cmplib.netlify.app/CSS/list.css">
+    <link rel="stylesheet" href="/CSS/header.css">
+    <!-- <link rel="stylesheet" href="/Pages/Product/product.css"> -->
```
Comment (anurag-kkumarr) [inline Pages/Login/login.html]: Import all these CSS into single file and import that file.

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]
Login button not working and Login page not opening. Fix that

File: Pages/Login/login.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Veekay Cart | Login</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link rel="stylesheet" href="https://veekay-cmplib.netlify.app/CSS/components.css">
```
Comment (ooreally) [inline Pages/Login/login.html:10]: use one css file.

**Review by ooreally — CHANGES_REQUESTED** [review]
PR and comment are totally different.
Not able to get what exactly is this PR. If this is for cart, can't find cart page here.  
Login page is not opening if thats wht you added.

File: Pages/Login/login.html (hunk 1 above)
Comment (vikrantkode) [reply Pages/Login/login.html]: Sure Will do that.....after completing whole...will definately rrefactor

**Comment by vikrantkode** [comment]
> PR and comment below it doesn't match. Didn't find any cart page in file change. PR raised should be for specific page you want to get reviewed. It will make reviewer's work easy.

Totally Agree....idk how i did this blunder . Next time i will make everything correct 200% sure while raising PR

**Comment by vikrantkode** [comment]
> PR and comment are totally different. Not able to get what exactly is this PR. If this is for cart, can't find cart page here. Login page is not opening if thats wht you added.

yeah it was a blunder from my-side. Will focus on this from next time

**Pull request merged**